CHANGELOG.md

```diff
@@ -1,49 +0,0 @@
-# Changelog
```

Comment:
Should the CHANGELOG be removed?

Reply:
Oops, I didn't mean to delete it, thank you for pointing it out!
Reverted: 103d45c